xmlStyle.go

```diff
@@ -211,8 +211,9 @@ func (styles *xlsxStyleSheet) getStyle(styleIndex int) *Style {
 			style.Alignment.Vertical = xf.Alignment.Vertical
 		}
 		style.Alignment.WrapText = xf.Alignment.WrapText
-
-		styles.Lock()
+        style.Alignment.TextRotation=xf.Alignment.TextRotation
+		
+        styles.Lock()
```

Comment:
Can you fix the indentation so that it matches the rest of the block that it is in?

Reply:
Ok! It's completed.